include/wil/wistd_memory.h

```diff
@@ -149,8 +149,8 @@ namespace wistd     // ("Windows Implementation" std)
     struct __second_tag {};
 
     template <class _T1, class _T2>
-    class __compressed_pair : private __compressed_pair_elem<_T1, 0>,
-                              private __compressed_pair_elem<_T2, 1> {
+    class __declspec(empty_bases) __compressed_pair : private __compressed_pair_elem<_T1, 0>,

```

Comment:
I've lost the pointer to the documentation on this, but without the `__declspec`, the size of the pair will be correct if one of the types is empty, however the address of `second` will still be beyond the bounds of the object